main.go

```diff
@@ -16,7 +16,7 @@ import (
 )
 
 // Version holds the current Gitea version
-const Version = "0.9.99.0915"
```

Comment:
What I do in other projects is always set the target version for each branch (ie: `<next version>-alpha`) and then separately and optionally append a changeset identifier which is generated on "make". As `Go` doesn't let us autogenerate things such setup would result in `<next version>-alpha` resulting in `go` obtained builds and `<next version>-alpha+<metadata>` resulting from builds via `make`.